baldrick/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 from baldrick.config import load, Config
 from baldrick.blueprints import github_blueprint, circleci_blueprint
+from baldrick.github.github_api import *  # noqa
```

Comment:
Actually I agree with Stuart that we should indeed import this into ``baldrick.github``, not ``baldrick``. It's not unconceivable that we could make baldrick work with other platforms in future?

Reply:
I can see me rage quitting github and moving to gitlab. Or wanting to use baldrick on my personal gitea instance ;)

Reply:
Okay, fair enough.

Reply:
Would be nice if `from baldrick import SomeHandler` is backend agnostic and something you can set as a preference but that is a battle for another day.